### PR TITLE
Fix app http error 500 when deploying on newer version of appengine

### PR DESCRIPTION
**Pull request by tadhunt**
Previously, this worked, but something changed recently such that now
if the package name for your service is 'main', and it has a main()
function, it works fine locally on the dev appserver, but fails with
error 500 when deployed to GAE.

This issue seems to have some relevant info:

https://code.google.com/p/googleappengine/issues/detail?id=13227

Anyway, the solution is to name the package something other than
'main', then it works fine.

I first tried simply removing the main() function, which made the
deployed app work, but breaks 'goapp build'.

tl;dr:  don't name your package 'main'.

### CODE REVIEW

**Comment by tadhunt** [comment]
@pmonkelban our policy is everything is reviewed, please take a look and "LGTM" if this looks ok to you then I'll merge it in.

I encountered this issue while trying to redeploy webhunt-dev.appspot.com with @speakerbug's CSS changes (so I could validate them), so suspect it'll fix any deployment issues you're having too...

**Comment by tadhunt** [comment]
I did author these commits

**Comment by tadhunt** [comment]
This PR has a bunch of cruft in it, so closing it.  It's replaced by a clean PR (https://github.com/google/web-bsd-hunt/pull/12)
